### PR TITLE
Added direct reservation link support

### PR DESCRIPTION
**Pull request by PasiVuohijoki**

### CODE REVIEW

File: src/directReservation/directReservationPage.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                        'direactReservation.errors.favouriteGenerationError',
+                        'Something went wrong! Try again later or contact the city official you received the direct reservation link from.'
+                      )}
+                    </Notification>
+                  )}
+                </>
+              ) : (
+                <>
+                  <p>
+                    {t(
```
Comment (EmiliaMakelaVincit) [inline src/directReservation/directReservationPage.tsx:82]: This flashes briefly before the link is processed even if you are already logged in.

Comment (PasiVuohijoki) [reply src/directReservation/directReservationPage.tsx:82]: hmm, I could not reproduce this? :thinking:

Comment (EmiliaMakelaVincit) [reply src/directReservation/directReservationPage.tsx:81]: Interesting, I'm seeing this every time I test this. It's even showing both the loader and the login message as long as the favourite hasn't been received and the redirect hasn't happened yet, I think.

image

Comment (PasiVuohijoki) [reply src/directReservation/directReservationPage.tsx:81]: ahh. now when logging out and then logging in again I get this also. All kind of other wonkiness is happening there also..

Comment (PasiVuohijoki) [reply src/directReservation/directReservationPage.tsx:68]: Now it is fixed. Duh, I am stupido.

**Review by EmiliaMakelaVincit — APPROVED** [review]
Looking good now!

**Pull request merged**